app/src/main/java/edu/uc/willi6jd/mycaloriecounter/ui/main/foodView/FoodActivity.kt

```diff
@@ -54,19 +54,16 @@ class FoodActivity : AppCompatActivity() {
 /**
  * This custom adapter is used to fill the data in the list view
  */
-private class foodCustomerAdapter(context: Context): BaseAdapter(){
-        private val mContext: Context
+private class FoodCustomerAdapter(context: Context): BaseAdapter(){
+        private val mContext: Context = context
 
-        //Test values for filling JSON
-        private val food = arrayListOf<String>("Peanut Butter", "Jelly", "Bread","Ham","Turkey")
```

Comment:
the use of <String> was not necessary since the compiler knows the list is built with strings.

Reply:
I lean toward keeping the generic identifier in... 1) to enforce that no one enters a non-string in the future, 2) to make a safe assumption that you're getting a string when you get something from this collection, and 2) for documentation.